internal/services/controller/controller_test.go

```diff
@@ -182,7 +183,7 @@ func TestController_ShouldReceiveDeltasBasedOnAvailableResources(t *testing.T) {
 								item.Data != nil &&
 								strings.Contains(string(*item.Data), expectedGVString) // Hacky but OK given this is for testing purposes.
 						})
-						require.True(t, found)
+						require.Truef(t, found, "missing object for %s %s", expectedGVString, expected.Kind)
```

Comment:
If there was an empty string, `%q` would produce an error msg that is easier to understand.

Reply:
I don't think it can be empty right now but replaced it. Maybe it will be some time or when something breaks.